docsets/Xojo/docset.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Xojo 2023r4",
+    "version": "1.0.0",
```

Comment:
This should be the version of the framework (i.e. use 2023r4 here). See https://github.com/Kapeli/Dash-User-Contributions/wiki/Docset-Versioning-Guidelines

Reply:
Continuing on with the prior comment train of thought, there needs to be some way of versioning the Docset. If I replace 1.0.0 with 2023r4, then how do you version the Docset when bugs are found or there are revisions? I'm happy to adjust things to your liking but just wanted to call out that Xojo's releases and documentation is actually quite different than much of the rest of the industry.

Reply:
From https://github.com/Kapeli/Dash-User-Contributions/wiki/Docset-Versioning-Guidelines:

* You should use the same version as the API the docset is for
* Anything that comes after a `/` inside the version is not displayed to the user at all. You can submit updates without changing the displayed version by adding anything after a `/`. For example, if the version you want to display to the user is `1.0.0`, you can still make Dash update a docset by setting the version to `1.0.0/whatever` and the user will only see `1.0.0`

So basically you'd set the version to `2023r4` (the version users see) and if you ever want to release an update you can add your own versioning after a `/`, e.g. `2023r4/whatever`